.kokoro/release/common.cfg

```diff
@@ -9,7 +9,7 @@ build_file: "gapic-generator-java/.kokoro/trampoline.sh"
 # Configure the docker image for kokoro-trampoline.
 env_vars: {
   key: "TRAMPOLINE_IMAGE"
-  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java11"
```

Comment:
Are we going to have a different release image for `gapic-generator-java` submodule?

Reply:
Decided that we are going to use a single Java11 image. But we should add release flag to gapic-generator-java or the parent if possible, to guarantee the compatibility of Java 8.